src/lightning/pytorch/trainer/call.py

```diff
@@ -82,7 +82,8 @@ def _call_setup_hook(trainer: "pl.Trainer") -> None:
 
     # Trigger lazy creation of experiment in loggers so loggers have their metadata available
     for logger in trainer.loggers:
-        _ = logger.experiment
+        if hasattr(logger, "experiment"):
```

Comment:
This should check if the attr exists before looping

Reply:
I don't understand. We need to loop over all loggers anyway.